go.mod

```diff
@@ -62,7 +65,7 @@ require (
 	github.com/bigkevmcd/go-configparser v0.0.0-20200217161103-d137835d2579
 	github.com/go-kit/kit v0.11.0
 	github.com/gobwas/glob v0.2.3
-	github.com/google/cadvisor v0.44.0
+	github.com/google/cadvisor v0.45.0-containerd-cri
```

Comment:
This is the official tracked containerd support. The code to get metrics is there but not turned on.